SecurityPkg/Library/DxeTcg2PhysicalPresenceMinimumLib/DxeTcg2PhysicalPresenceMinimumLib.inf

```diff
@@ -9,7 +9,7 @@
 #
 ##
 
-#Override : 00000002 | SecurityPkg/Library/DxeTcg2PhysicalPresenceLib/DxeTcg2PhysicalPresenceLib.inf | 6205753f2abf9126f2711c5f154f9f99 | 2024-07-24T18-06-30 | 69ff4b6fc889c8e66101cad3dcf8be3d516e038d
+#Override : 00000002 | SecurityPkg/Library/DxeTcg2PhysicalPresenceLib/DxeTcg2PhysicalPresenceLib.inf | 0e330f45cfc28a0ec3ac6fccf5985f97 | 2024-07-30T05-15-01 | 185028e6eef7f50c471e2ca0b6a31a40ce408d6e
 # This is not a true override, but spell changes to ensure mu_tiano_plus passes CI is required and changes the hash.

```

Comment:
Why do we treat this as an override instead of just a spelling update?

Reply:
Removing the comment. It is a true derivative of the original file.